scout/server/blueprints/variant/templates/variant/acmg.html

```diff
@@ -29,22 +29,27 @@
     <form action="{{ url_for('variant.variant_acmg', institute_id=institute._id, case_name=case.display_name, variant_id=variant._id) }}" method="POST">
       <div class="card panel-default mt-3">
         <div class="card-body">
-          {% if not evaluation %}
-              <div class="mt-3 fixed-bottom bg-light border">
-                <div class="text-center">
-                  {% for option in ACMG_OPTIONS %}
-                    <a id="acmg-{{ option.code }}" class="btn acmg-preview">{{ option.label }}</a>
-                  {% endfor %}
-                </div>
-              </div>
-            <button class="btn btn-primary form-control">Submit</button>
-          {% else %}
-            <h4>
+          {% if evaluation %}
+             <h4>
               {{ evaluation.classification.label }}
               <span class="badge bg-info">{{ evaluation.classification.short }}</span>
             </h4>
             By {{ evaluation.user_name }} on {{ evaluation.created_at.date() }}
+            {% if edit %}
+              <br><br>
+              <button class="btn btn-primary form-control">Update</button>
```

Comment:
I really like that we give the users the feature to continue based off a given evaluation, while still preserving the audit log. While edit is a useful short name, this is not really an edit, but rather a cloning or re-classification. And in particular clicking update doesn't really update, rather create a new event. How about just renaming this button to indicate that; say "reclassify", "resubmit", or "add"?
```suggestion
              <button class="btn btn-primary form-control">Reclassify</button>
```

Reply:
Good suggestion, applied!

Reply:
Added tooltips as well:

<img width="452" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/b35b450a-e4a7-4c50-98d9-982045b539f1">

<img width="1110" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/c1d3a527-6a8d-436e-a97d-c98bbc59a8f0">